examples/db-templates/mariadb-ephemeral-template.json

```diff
@@ -32,6 +32,10 @@
                 "name": "${DATABASE_SERVICE_NAME}"
             },
             "stringData": {
+                "database-type": "mysql",
+                "database-uri": "mysql://${DATABASE_SERVICE_NAME}:3306",
+                "database-host": "${DATABASE_SERVICE_NAME}",
+                "database-port": "3306",
```

Comment:
any reason not to add these to the list of things being exposed above?

Reply:
(the uri one i realize is already exposed below, but you could expose the host, port, and type.  you could also move the uri exposure up here so they are all lumped together, though it would mean changing the uri value from an ip to the service hostname, but that should be fine i would think)